tests/storage/cdi_clone/test_clone.py

```diff
@@ -51,9 +51,9 @@ def create_vm_from_clone_dv_template(
     with VirtualMachineForTests(
         name=vm_name,
         namespace=namespace_name,
-        os_flavor=OS_FLAVOR_CIRROS,
+        os_flavor=OS_FLAVOR_FEDORA,
         client=client,
-        memory_requests=Images.Cirros.DEFAULT_MEMORY_SIZE,
+        memory_requests=Images.Fedora.DEFAULT_MEMORY_SIZE,
```

Comment:
`memory_requests` field is deprecated, please use `memory_guest` instead

```suggestion
        memory_guest=Images.Fedora.DEFAULT_MEMORY_SIZE,
```